pom.xml

```diff
@@ -129,6 +129,8 @@ Universidad Carlos III de Madrid.</license.copyrightOwners>
 
 		<protobuf.version>3.2.0</protobuf.version>
 		<tensorflow.version>1.15.0</tensorflow.version>
+		<jna.version>5.11.0</jna.version>
+		<imagej-tensorflow.version>1.1.6</imagej-tensorflow.version>
```

Comment:
Ahh, sorry @mkitti, it's been awhile since I've had my head in the ImageJ-TensorFlow space. I conflated the versions of TensorFlow and ImageJ-TensorFlow. Ignore my noise, then.

Reply:
Let me know if there is anything else to do here.

Reply:
Let me know if anything else needs to be done here.